x-pack/plugins/ml/public/application/explorer/components/explorer_no_jobs_found/explorer_no_jobs_found.js

```diff
@@ -25,7 +25,7 @@ export const ExplorerNoJobsFound = () => (
       <EuiButton color="primary" fill href="ml#/jobs">
         <FormattedMessage
           id="xpack.ml.explorer.createNewJobLinkText"
```

Comment:
Fix `eslint` (part 1/2)
```suggestion
        <FormattedMessage id="xpack.ml.explorer.createNewJobLinkText" defaultMessage="Create job" />
```